### PR TITLE
meek/basic UI styling

### LINKED ISSUE #65: Basic UI styling
## Acceptance Criteria

- [ ] Match UI styling from Figma design

ui-figma-march-29

### PR DESCRIPTION
**Pull request by mmsaki**
## Description

- Match style closely to what is specified in Figma frames.
- Responsive design for narrow screen
- Added images
- Added starting fonts

## Related Issue

Closes #65 #66

## Acceptance Criteria

- [x] Match Figma design
- [x] Fix responsive issues

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓  | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |

### Before

Screenshot 2023-03-29 at 12 07 29 PM

### After

Screenshot 2023-03-29 at 12 42 48 PM

### Next steps

- Improve design to a more playful colorful and gameish.
- Feedback and updates can be added on figma

### CODE REVIEW

File: app.js
Comment (terrifricker) [inline app.js:1]: In the future, formatting changes should be isolated into their own pull request.  It was very difficult to see where actual changes in the code were made.  I had to go line by line, every line, and visually inspect for changes.  Sometimes the sections were long enough that I was not able to see the lines on the same screen even.

File: data/data.json
Comment (terrifricker) [inline data/data.json:1]: Again, this should have been in a separate formatting change pull request.  It is impossible to know if a single character was changed in this file.

File: instructions.html [hunk 1]
```diff
@@ -1,39 +1,48 @@
...
+		<link rel="stylesheet" href="instructions.css" />
+		<link rel="stylesheet" href="styles.css" />
+	</head>
+	<body>
+		<div class="wrapper">
+			<header class="nav-bar">
+				<div onclick="document.location='index.html'">
+					<h1>MAD LIBS</h1>
+				</div>
+				<a href="play.html" onclick="document.location='play.html'">Play</a>
```
Comment (terrifricker) [inline instructions.html]:
```suggestion
				<a href="play.html">Play</a>
```

File: ourteam.html [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
+		<link rel="stylesheet" href="styles.css" />
+		<link rel="stylesheet" href="ourteam.css" />
+	</head>
+	<body>
+		<div class="wrapper">
+			<header class="nav-bar">
+				<div onclick="document.location='index.html'">
+					<h1>MAD LIBS</h1>
+				</div>
+				<a href="play.html" onclick="document.location='play.html'">Play</a>
```
Comment (terrifricker) [inline ourteam.html]:
```suggestion
				<a href="play.html">Play</a>
```

**Review by terrifricker — COMMENTED** [review]
I've started going through, but I'm not on my own computer with VSCode available to work through the major css.  It will be nice to have it all consistent.

File: instructions.html (hunk 1 above)
Comment (mmsaki) [reply instructions.html]: thanks for that

File: data/data.json
Comment (mmsaki) [reply data/data.json:1]: no content has changed on this file i think my formatter might have picked up some linting and spacing issues and auto-formatted

File: app.js
Comment (mmsaki) [reply app.js:1]: @terrifricker Hey thanks for the feedback. I agree! Formatting does make it hard to review important changes. Githib actually has some nice features to hide white space formatting and settings to make that easy for you. [See this article](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/committing-and-reviewing-changes-to-your-project). Here's the tip I got from the article 

image

Comment (terrifricker) [reply app.js:1]: Thank you!!

Comment (terrifricker) [reply app.js:1]: Argh!  I don't use desktop.  Guess I need to change that.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@600&display=swap');
+
+/* 
+* Begin styling for html native elements
+*/
+
+* {
+	display: flex-box;
```
Comment (terrifricker) [inline styles.css]:
```suggestion
```

Comment (terrifricker) [reply styles.css]: flex-box is not a display option, and you wouldn't want this on * global anyway.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+h2 {
+	font-size: 2rem;
+}
+
+h3 {
+	font-size: 1.2rem;
+}
+
+html body {
```
Comment (terrifricker) [inline styles.css]: The selector html body means the body element within html which is just the body element.  html is not needed.
```suggestion
body {
```

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+.wrapper {
+	min-height: 100vh;
+	display: grid;
+	grid-template-rows: auto 1fr auto;
+}
+
+.logo {
+	height: 100px;
+	text-align: center;
```
Comment (terrifricker) [inline styles.css:70]:
```suggestion
```
not need now that .logo is an image.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+.flex-menu {
+	height: 500px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+}
+
+.buttons {
+	background-color: rgb(255, 255, 255, 0);
```
Comment (terrifricker) [inline styles.css:103]: why not background-color white or not defined instead of transparent white?

**Review by terrifricker — CHANGES_REQUESTED** [review]
We need to work on the media query.  Here are the images I have on Chrome Dev tools for a screen width of 360px.  Either a hamburger menu or flex-direction column as a start.

Screen Shot 2023-03-29 at 10 25 55 PM
Screen Shot 2023-03-29 at 10 26 05 PM
Screen Shot 2023-03-29 at 10 26 18 PM

File: data/data.json
Comment (terrifricker) [reply data/data.json:1]: What are you using as a formatter/linter?  I use Prettier and ESLint.

File: styles.css (hunk 3 above)
Comment (mmsaki) [reply styles.css]: I can look into that. Great catch! I think I meant to put 
```css
box-sizing: border-box;
```

**Comment by mmsaki** [comment]
@terrifricker I was aiming for 390px screen width to start with. I agree that a hambager menu. Only reason I didn't add a hamburger menu on is PR is because it has not been specified on the design yet. Thank you for catching that. Also, the browser's sometimes resize with extreme zoom out not sure exactly how to fix that, but you should notice the same thing for other websites on your second image.

**Review by terrifricker — APPROVED** [review]

**Review by terrifricker — APPROVED** [review]
Woo hoo! 

Approved!

**Comment by terrifricker** [comment]
>  Also, the browser's sometimes resize with extreme zoom out not sure exactly how to fix that, but you should notice the same thing for other websites on your second image.

You're exactly right!  How weird!

**Pull request merged**